tracer/test/Datadog.Trace.Tests/Telemetry/TelemetrySettingsTests.cs

```diff
@@ -274,5 +276,20 @@ public void SetsTelemetryEnabledBasedOnAgentlessEnabledAndAgentProxyEnabled(bool
                 settings.Agentless.Should().BeNull();
             }
         }
+
+        [Theory]
+        [InlineData("0", 60)]
+        [InlineData(null, 60)]
+        [InlineData("", 60)]
+        [InlineData("invalid", 60)]
+        [InlineData("3600", 3600)]
```

Comment:
Maybe add a "mid value" one here too, just for completeness
```suggestion
        [InlineData("invalid", 60)]
        [InlineData("523", 523)]
        [InlineData("3600", 3600)]
```